### DISCUSSION TITLE
Ticket #5315 Title: Update API Documentation - Rate Limiting Section

### DISCUSSION THREAD

**Discussion by sammohdnak** [community/community#147676] [npm]
Ticket #5315
Title: Update API Documentation - Rate Limiting Section
Priority: Medium
Description: Need to review and update the rate limiting documentation to include recent changes in API behavior

**Comment by ebndev** [comment]
Hi @sammohdnak, thanks for being a part of the GitHub Community!

Can you share more about what feedback you'd like to leave? Giving a few more details might help someone give you a nudge in the right direction. 😄
